### PR TITLE
Fax office immediately after applicant indicates they are done

### PR DESCRIPTION
**Pull request by zspencer**
### Why? 
Waiting for a fax is not a great user experience when demoing.

### CODE REVIEW

File: app/controllers/standard_steps_controller.rb [hunk 1]
```diff
@@ -18,6 +21,14 @@ def update
 
   private
 
+  def after_successful_update
+
```
Comment (adarsh) [inline app/controllers/standard_steps_controller.rb]: Can you put a `#noop` comment in here to just clarify it's emptiness? 🕳

File: app/controllers/success_controller.rb [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
-
   def previous_path(*_args)
     nil
   end
 
+  def next_path
+    root_path(anchor: "fold")
+  end
+
   private
```
Comment (adarsh) [inline app/controllers/success_controller.rb]: Style: newline after `private`

File: app/controllers/standard_steps_controller.rb [hunk 3]
```diff
@@ -3,13 +3,16 @@
 class StandardStepsController < StepsController
   def edit
     @step = step_class.new(existing_attributes.slice(*step_attrs))
+
+    before_rendering_edit
```
Comment (adarsh) [inline app/controllers/standard_steps_controller.rb]: Naming: `before_rendering_edit` > `before_rendering_edit_action`? or `before_edit_action`?

File: app/controllers/standard_steps_controller.rb (hunk 1 above)
Comment (zspencer) [reply app/controllers/standard_steps_controller.rb]: Yep!

File: app/controllers/standard_steps_controller.rb [hunk 4]
```diff
@@ -3,13 +3,16 @@
...
   end
 
   def update
     @step = step_class.new(step_params)
 
     if @step.valid?
       current_snap_application.update!(step_params)
+      after_successful_update
```
Comment (adarsh) [inline app/controllers/standard_steps_controller.rb]: Naming: `after_successful_update` > `after_successful_update_action`

File: app/controllers/standard_steps_controller.rb [hunk 5]
```diff
@@ -21,8 +21,13 @@ def update
 
   private
 
+  # Hook for step controllers to add behavior after successful update. This is
+  # intentionally a no-op in the StandardStepsController.
```
Comment (adarsh) [inline app/controllers/standard_steps_controller.rb]: I'd prefer the method name be clearer than we add comments. 

What about `after_successful_update_hook`?

Comment (zspencer) [reply app/controllers/standard_steps_controller.rb]: Done.

File: app/controllers/success_controller.rb (hunk 2 above)
Comment (zspencer) [reply app/controllers/success_controller.rb]: As in there shouldn't be one?

**Pull request merged**